modules/rosa-accessing-your-cluster.adoc

```diff
@@ -42,7 +47,7 @@ Any optional fields can be left empty and a default will be selected.
 ----
 +
 .. Follow the URL from the output. This creates a new OAuth application in the GitHub organization you specified.
-.. Click *Register applicaton* to access your Client ID and Client Secret.
+.. Click *Register application* to access your Client ID and Client Secret.
```

Comment:
Are _Client ID and Client Secret_ both capitalized? Or does lowercase make more sense here?

Reply:
Good question. I inherited this topic, but since it seems to be a noun with the use of the possessive, I believe it ought to be lower case. Thanks for catching that.